clang/lib/Headers/cheri.h

```diff
@@ -146,27 +146,18 @@ void * __capability cheri_program_counter_get(void) {
 /* TODO: Should these be builtins to get better diagnostics? */
 
 static inline __attribute__((always_inline)) __attribute__((warn_unused_result))
-__SIZE_TYPE__
```

Comment:
This header was being used by the initial QtQml port, so it started triggering warnings.